### DISCUSSION TITLE
What is the dataset update frequency?

### DISCUSSION THREAD

**Discussion by mortgad** [dcaribou/transfermarkt-datasets#214] [FAQ] [answered]
Hi David,

What is the reason the latest gameweek has not been added to the dataset? 
The latest entries in the appearances.csv file are from the 20th of September.

Kind regards
Morten

**Comment by dcaribou** [comment]
Hey @mortgad,

The last pipeline run blocked on the [testing step](https://github.com/dcaribou/transfermarkt-datasets/actions/runs/6308184075/job/17125965291). When data checks fail, the pipeline stops automatically so that invalid data does not flow into Kaggle.

In this case, apparently, the row count for `player_valuations` dropped by 20%, hence the pipeline stopped.
> 04:51:43  Completed with 1 error and 0 warnings:
04:51:43  
04:51:43  Failure in test dbt_expectations_expect_table_row_count_to_be_between_player_valuations_460000__430000 (models/curated/models.yml)
04:51:43    Got 1 result, configured to fail if != 0

I'll have a look. In the meantime, do you need `player_valuations` itself?  All the other files (games, appearances, etc,) seem fine, so I can get those updated.

**Reply by mortgad** [reply]
Okay, I think updating the other files is fine for now.
So all that means is that new valuations from the last week are not available?

**Reply by dcaribou** [reply]
> So all that means is that new valuations from the last week are not available?

Yes, exactly

Cool. I've unblocked it now for anything that is not `player_valuations`. It should be visible in Kaggle in a few minutes.

Also, I've created #215 to track the fix for the `player_valuations`, feel free to subscribe.

**Comment by dcaribou** [comment, marked as answer]
To answer the question on update frequency ~~the project currently runs on a weekly basis every Tuesday night (CET), however this schedule is subject to adjustments.~~

The schedule is defined in the [on-schedule.yml](https://github.com/dcaribou/transfermarkt-datasets/blob/master/.github/workflows/on-schedule.yml) workflow from github actions. Refer to that file for the most updated schedule definition.

**Reply by mortgad** [reply]
Okay, good to know!
Would there be any issue with two weekly updates?
I am thinking of the weeks with midweek games, fx the current week.
For these weeks, two updates, e.g. Tuesday 04:00 and Friday 04:00, would make sure that before most fixtures the latest played game is present in the dataset.

**Reply by dcaribou** [reply]
Hey @mortgad,
That sounds reasonable actually. Let me add that.

**Reply by dcaribou** [reply]
There you go https://github.com/dcaribou/transfermarkt-datasets/commit/132a9930eae07d419b25d2433430a4c0e1467d64

**Reply by mortgad** [reply]
Awesome! Thank you David
